crates/parser/src/grammar/generic_args.rs

```diff
@@ -52,7 +52,6 @@ fn generic_arg(p: &mut Parser) {
                     // NameRef::, this is a path type
                     path_seg.complete(p, PATH_SEGMENT);
                     let qual = path.complete(p, PATH);
-                    opt_generic_arg_list(p, false);
```

Comment:
@Veykril could you take  a look here as well? I *think* this is not needed, as we parse opt generics *before* match alraedy.

Reply:
Ye I think you are right